extensions/wikia/MercuryApi/tests/MercuryApiModelTest.php

```diff
@@ -12,19 +12,21 @@ public function setUp() {
 	 */
 	private function setupGlobals() {
 		$this->mockGlobalVariable('wgLoadAdsInHead', true);
+		$this->mockGlobalVariable('wgAdDriverBottomLeaderboardImpressionCapping', false);
```

Comment:
Thanks for your input! 

This variable has value set in DefaultSettings.php - array value. 
Also in WF there is variable of Array type and you can't fill it with falsy value. So probably it's not worth to do this